app/Console/Kernel.php

```diff
@@ -28,6 +32,87 @@ protected function schedule(Schedule $schedule)
     {
         $schedule->command('telescope:prune')->daily();
         // $schedule->command('sync:tg-forum-groups')->dailyAt('04:00');
+
+        $schedule->command('sys:statistics:daily')
+            ->dailyAt('00:01')
+            ->onFailure($this->failureMessage('Syncing of daily statistics has failed (sync:statistics:daily)'))
+            ->emailOutputOnFailure($this->failureEmail);
+
+        $schedule->command('sync:helpdesk')
```

Comment:
Helpdesk should already be covered - ref https://github.com/VATSIM-UK/core/blob/master/app/Listeners/Mship/SyncSubscriber.php#L24

Reply:
removed!